src/components/app/custom-els/FileDrop/styles.css

```diff
@@ -0,0 +1,22 @@
+file-drop {
+  display: block;
+  overflow: hidden;
+  touch-action: none;
+  height:100%;
+  width:100%;
+}
+
+.drop-active {
+}
+
+.drop-valid {
+  transition: opacity 200ms ease-in-out, background-color 200ms;
+  opacity: 0.5;
+  background-color:green;
+}
+
+.drop-invalid {
+  transition: opacity 200ms ease-in-out, background-color 200ms;
+  opacity: 0.5;
+  background-color:red;
+}
```

Comment:
Same issues as before.

Reply:
What issue? I don't see it in the previous comments.

Reply:
I also can't find it. Hmm.

Anyway, these are Squoosh styles. The component itself probably shouldn't have any styles. Move the squoosh styles out of custom-els and into `app/styles.scss`.

Reply:
done.